Story-Mode-Roadmap.md

```diff
@@ -0,0 +1,188 @@
+  - When beat, the Yeti accidentally informs Tux on the crystal mines and a boat to use to sail up north
+  - The sun sets when Tux arrives at the crystal mines and starts sailing north during the night
+
+**Rooted Forest**
+  - Tux arrives at sunrise
+  - First half of Rooted Forest is Tux exploring the new land, looking for clues on where the next castle is located
+  - Tux passes friendly Granito species along his way
+  - When on a higher section of the forest, Tux takes notice of a big tower towards the far east
+  - During a heavy thunderstorm Tux seeks shelter in a dark cave, stumbling into another outpost
+  - When Tux left the cave the next day dawned and Tux decided to take a small nap
+  - An ill Vicious Ivy peaks Tux's curiosity, followed by a bunch of purple roots bursting out of the ground everywhere around him
+  - Tux makes his way towards the tower while avoiding the roots as they over take the forest
+  - When entering the castle and reaching the very top, the roots attempt to fully destroy the tower and bury Tux
+  - Tux escapes barely but runs into the Ghost Tree, initiating the fight
+  - When beat, the Ghost Tree gets destroyed and the land slowly begins to heal
+  - Surviving Granito thank Tux and wish to help Tux on his journey, leading him to the tropical island Nolok had headed.
+
+---
+
+## Future Story Draft (post-v0.7)
+
+This section is an early and still unfinished draft of the continuation of story mode past Rooted Forest.
+Note that all you see here is but an idea of what is to come story-wise and may differ from whatever the final execution may be!
+
+**Tropical Paradise**
+  - Tux arrives at night at a small beach isle
+  - Tux must swim from there to the main island
+  - More and more Totems appear on the island the closer and closer Tux gets to the island's temple
+  - Toucan is found inside their throne room admiring themselves on a painting
+  - Toucan is infuriated by Tux's sudden appearance, leading him outside, initiating the fight
+  - When beat, Toucan flees the scene with Tux following suit in hopes to finally found out about Nolok's and Penny's whereabouts
+  - Tux follows Toucan to a hidden fort with a teleporter
+  - Nolok is on his way to activate the teleporter
+  - Upon noticing Tux, Nolok pushes Penny into the teleporter as he summons purple roots to block Tux's path, using the teleporter himself soon after
+  - Tux fights himself through the fort until he manages to reach and use teleporter to follow suit
+
+**Nolok's Mountain Peak**
+  - Tux finds himself on top of a huge mountain
+  - There is a massive fortress in the center of the mountain's peak
+  - The entrance door is locked by five colored keys
+
+> If Tux has not all five NOLOK-keys collected:
+
+  - Nolok tells Tux to bring all the five hidden keys spread through out Icy Island, Rooted Forest and Tropical Paradise
+
+> If Tux has all five NOLOK-keys collected:
```

Comment:
NOLOK keys

Reply:
I think it does not matter in this case, does it not?